### DISCUSSION TITLE
I asked for a refund but haven't received a response from the GitHub team.

### DISCUSSION THREAD

**Discussion by Mo-bito** [community/community#128310] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, I asked the GitHub team to cancel my monthly plan for Copilot. I forgot to cancel it myself, and yesterday I was surprised by a bill for the subscription. I tried to submit a ticket but haven't received a response from the GitHub team yet. When can I expect them to answer, and is there any other way to request a refund using the system?

**Comment by ebndev** [comment]
Hi @Mo-bito,

Please see this previous answer to [self-serve refunds for Copilot](https://github.com/orgs/community/discussions/45645#discussioncomment-5789812). Please comment there if you have any issues so the moderator there can answer if they are able to.

As no other Community Discussions team members will be able to provide additional help with account related questions, I am going to close this discussion. We appreciate your understanding!
